docker-entrypoint.sh

```diff
@@ -23,6 +23,11 @@ if [[ "$APPLY_MIGRATIONS" = "1" ]] && [[ "$SEED_DEVELOPMENT_DATA" = "1" ]]; then
     ./manage.py seed_development_data
 fi
 
+if [[ "$APPLY_MIGRATIONS" = "1" ]] && [[ "$SET_ALLOWED_DATA_FIELDS" = "1" ]]; then
```

Comment:
Not sure why this should be tied to the value of `APPLY_MIGRATIONS`? 🤔

Reply:
APPLY_MIGRATIONS has been removed  from the condition